iDiff/utils/docker_utils.go

```diff
@@ -0,0 +1,172 @@
+package utils
```

Comment:
If possible to refactor code to move some of the stuff in image_utils that centers around Docker into this file, I think that would be better organization...